### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by joychugh** [j-e-k/pgoapi#20]
Show nearby pokemon in the Map

**Comment by j-e-k** [comment]
Hmmm, do you mean where they have been encountered or where they will be encountered?

The former is easy, the latter is slightly more complicated because it would involve spiraling out with get_map_objects, which is more likely to result in a softban.....What do you think?

**Comment by joychugh** [comment]
@j-e-k yes wherre they have been encountered. when we do get map objects, just plant them on a map.

**Comment by j-e-k** [comment]
Ok will do this later today
